### DISCUSSION TITLE
Nvim autocompletion - How to make snippet appear after fields

### DISCUSSION THREAD

**Discussion by tinwritescode** [NvChad/NvChad#1649] [Q&A] [answered]
![Screen Shot 2022-10-24 at 12 19 25](https://user-images.githubusercontent.com/40380704/197453471-f51591c4-d495-4114-957c-f81ebabcb6b2.png)

Hi guys, thank you for such a useful nvim starter base for getting started.

I have a question when I activate autocompletion using `ctrl + space`, the dialog shows up but it shows snippets before fields, I want fields to show up first. How can I?

**Comment by siduck** [comment, marked as answer]
@tinwritescode  check the cmp config's source table and then override cmp config
